ir_action_mail_translate/models/ir_actions_server.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2015 Savoir-faire Linux (<http://www.savoirfairelinux.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+import time
+from openerp.osv import orm
+
+
+class IrActionsServer(orm.Model):
+
+    _inherit = 'ir.actions.server'
+
+    def merge_message(self, cr, uid, keystr, action, context=None):
+        """
+        Translate the message in the recipient's language
+        """
+        if context is None:
+            context = {}
+
+        # If the recipient is not found, take the admin user instead
+        recipient = self.pool['res.users'].browse(
+            cr, uid, uid, context=context)
+
+        active_id = context.get('active_id')
+        active_model = context.get('active_model')
+
+        if active_id and active_model:
+            obj_pool = self.pool[active_model]
+            res = obj_pool.browse(
+                cr, uid, active_id, context=context)
+
+            # Get the address of destination of the action
+            cxt = {
+                'self': obj_pool,
+                'object': res,
+                'obj': res,
+                'pool': self.pool,
+                'time': time,
+                'cr': cr,
+                'context': dict(context),
+                'uid': uid,
+                'user': recipient,
+            }
+
+            try:
+                address = eval(str(action.email), cxt)
```

Comment:
OK. Just use safe_eval, because that is what ir_actions.py uses (see imports in that file).